common/src/backend/java/dev/engine_room/flywheel/backend/gl/GlCompat.java

```diff
@@ -215,4 +215,17 @@ void main() {}
 
 		return success;
 	}
+
+	static class Workarounds {
+		// Intel HD Graphics support* instancing and indirect rendering.
+		// * However, in reality the support is horrible and doesn't actually work
+		// so whenever we encounter these, we have to fall back to flywheel:off
+		public static boolean isIntelHDGraphics() {
+			if (DRIVER != Driver.INTEL)
```

Comment:
could we bake this into the driver enum instead? it's kinda used as "vendor" right now but it is intended for this kind of filtering